src/odemis/driver/omicronxx.py

```diff
@@ -385,18 +385,18 @@ def _setValue(self, com, val=None):
         ans = self.acc.sendCommand("%s%s%s" % (com, self._com_chan, val))
         if not ans.startswith(com):
             raise IOError("Expected answer to start with %s but got %s" %
-                          (com, ans.encode('string_escape')))
+                          (com, ans))
```

Comment:
All the following lines should have "ans" escaped, as we don't know what it contains.